### DISCUSSION TITLE
ST7789 GND VCC picture

### DISCUSSION THREAD

**Discussion by alexanderbehrens1978** [frntc/Sidekick64#74] [Q&A]
Hi,
what is the right GND VCC Combination for the ST7789? 
I leave the solderpads blank. No Screen on the ST7789.
But anything else works.
Please post a picture with the right solder combination on the GND VCC fields.

**Comment by Shadowtrance** [comment]
That will depend on **your** particular screen as the pinout can be different at times, which is why those jumper pads exist.
On the screens i bought the pinout is as follows (looking down at the screen)
GND VCC SCK SDA RES DC BLK
First jumper (left side) is set with the left and middle linked and the 2nd jumper is set with the right and middle linked.
Again looking down at the screen/pcb.
Hope that helps. :)

**Comment by frntc** [comment]
here's a nice explanation: https://www.forum64.de/index.php?thread/101221-sidekick64-aufbau-probleme-lösungen/&pageNo=60
